ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -0,0 +1,212 @@
+            process_variables[0].get().getNumberOfComponents(),
+            DisplacementDim);
+    }
+
+
+    // Constitutive relation.
+    // read type;
+    auto const constitutive_relation_config =
+        //! \ogs_file_param{process__HYDRO_MECHANICS_constitutive_relation}
+        config.getConfigSubtree("constitutive_relation");
+
+    auto const type =
+        constitutive_relation_config.peekConfigParameter<std::string>("type");
+
+    std::unique_ptr<MaterialLib::Solids::MechanicsBase<DisplacementDim>>
+        material = nullptr;
+    if (type == "LinearElasticIsotropic")
+    {
+        material =
+            MaterialLib::Solids::createLinearElasticIsotropic<DisplacementDim>(
+                parameters, constitutive_relation_config);
+    }
+    else
+    {
+        OGS_FATAL(
+            "Cannot construct constitutive relation of given type \'%s\'.",
+            type.c_str());
+    }
+
+    // Intrinsic permeability
+    auto& intrinsic_permeability = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__HYDRO_MECHANICS_intrinsic_permeability}
+        "intrinsic_permeability",
+        parameters, 1);
+
+    DBUG("Use \'%s\' as hydraulic conductivity parameter.",
+         intrinsic_permeability.name.c_str());
+
+    // Storage coefficient
+    auto& storage_coefficient = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__HYDRO_MECHANICS_storage_coefficient}
+        "storage_coefficient",
+        parameters, 1);
+
+    DBUG("Use \'%s\' as storage coefficient parameter.",
+         storage_coefficient.name.c_str());
+
+
+    // Fluid viscosity
+    auto& fluid_viscosity = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__HYDRO_MECHANICS_fluid_viscosity}
+        "fluid_viscosity",
+        parameters, 1);
+    DBUG("Use \'%s\' as fluid viscosity parameter.",
+         fluid_viscosity.name.c_str());
+
+    // Biot coefficient
+    auto& biot_coefficient = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__HYDRO_MECHANICS_biot_coefficient}
+        "biot_coefficient",
+        parameters, 1);
+    DBUG("Use \'%s\' as Biot coefficient parameter.",
+         biot_coefficient.name.c_str());
+
+    // Porosity
+    auto& porosity = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__HYDRO_MECHANICS_porosity}
+        "porosity",
+        parameters, 1);
+    DBUG("Use \'%s\' as porosity parameter.",
+         porosity.name.c_str());
+
+    // Solid density
+    auto& solid_density = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__HYDRO_MECHANICS_solid_density}
+        "solid_density",
+        parameters, 1);
+    DBUG("Use \'%s\' as solid density parameter.",
+         solid_density.name.c_str());
+
+    // Fluid density
+    auto& fluid_density = findParameter<double>(
```

Comment:
I'd like to skip the usage of fluid properties for now. When the material lib is set into a general framework we can start updating the processes to use that framework.